src/pykeen/models/unimodal/conv_e.py

```diff
@@ -291,15 +291,15 @@ def _convolve_entity_relation(self, h: torch.LongTensor, r: torch.LongTensor) ->
 
         try:
             # batch_size, num_input_channels, 2*height, width
-            if self.apply_batch_normalization:
+            if self.apply_batch_normalization and batch_size > 1:
```

Comment:
can you save `batch_size > 1` into a variable so there's some implicit documentation? It also reduces the need to test the condition 3 times